netkan/netkan/cli/common.py

```diff
@@ -14,7 +14,8 @@
 def ctx_callback(ctx: click.Context, param: click.Parameter,
                  value: Union[str, int]) -> Union[str, int]:
     shared = ctx.ensure_object(SharedArgs)
-    setattr(shared, param.name, value)
+    if param.name:
+        setattr(shared, param.name, value)
```

Comment:
What change required this?

Reply:
I think that was this:

https://github.com/KSP-CKAN/NetKAN-Infra/runs/2588300928

```
_____________________________ netkan/cli/common.py _____________________________
 note: In function "ctx_callback":
17:21: error: Argument 2 to "setattr" has incompatible
```

Apparently `param.name` isn't guaranteed not to be `None`.